src/Components/WeatherDetails/WeatherDetails.js

```diff
@@ -1,67 +1,107 @@
-import React from 'react';
+import React, { Component } from 'react';
 
+import Button from './WeatherShowMoreLessButton';
+import { FadeInTop } from '../../common/Animations/Animations';
+import WeatherAdditionalDetails from '../WeatherDetails/WeatherAdditionalDetails';
 import WeatherIcon from '../WeatherIcon/WeatherIcon';
 
-const WeatherDetails = ({ data }) => {
-  const { cod } = data;
-  const locationNotFound = cod !== 200 ? true : false;
+class WeatherDetails extends Component {
+  state = {
+    showAdditionalDetails: false,
+    active: false,
+  };
 
-  if (locationNotFound) {
-    return (
-      <div className="weather-details">
-        <div className="weather-details__message alert alert-danger">
-          Sorry, we didn't find the location
-        </div>
-      </div>
-    );
+  handleAdditionalDetailsState = () => {
+    this.setState({
+      showAdditionalDetails: !this.state.showAdditionalDetails,
+    });
+  };
+
+  componentDidMount() {
+    this.setState({ active: true });
   }
 
-  const {
-    name,
-    main: { humidity, pressure, temp, temp_max, temp_min },
-    wind: { speed },
-    sys: { country },
-    weather: [{ icon, desctiption }],
-  } = data;
+  render() {
+    const { cod } = this.props.data;
+    const locationNotFound = cod !== 200 ? true : false;
```

Comment:
unnecessary ternary operator.
I would go this way:

`const locationFound = cod === 200;`

if code will be 200 you will get TRUE, if code will be 400 404 etc, you will get FALSE :)